packages/payment-processor/src/payment/settings.ts

```diff
@@ -52,4 +52,15 @@ export interface IRequestPaymentOptions {
   conversion?: IConversionSettings;
   /** Optional, enable to approve only specific amount of token. Defaults to MAX_ALLOWANCE if not set */
   approval?: IApprovalSettings;
+
+  /** Optional, specifies if escrow is being used */
+  isEscrow?: boolean;
```

Comment:
added here because "isEscrow?: boolean;" is defined in the interface replicated within invoicing repository